TestSuite/pack.py

```diff
@@ -216,14 +216,17 @@ def create_integration(
                 "commonfields": {"id": name, "version": -1},
                 "name": name,
                 "display": name,
-                "description": f"this is an integration {name}",
+                "description": description
+                if description is not None
+                else f"this is an integration {name}",
                 "category": "category",
                 "script": {
                     "type": "python",
                     "subtype": "python3",
                     "script": "",
                     "commands": [],
                 },
+                "configuration": [],
```

Comment:
what's this change?

Reply:
The issue with the unit-test on `validate_integration_test.py` I've updated, was that the example `old_integration` that was generated was invalid. Since this PR adds validation to old files, the test failed (it wasn't checked before).

While debugging, one of the validation errors that were found on the example `old_integration` was that the `configuration` key was missing, so it appears to be required in valid integrations' YAML file.

Because of that, it seemed to make more sense to add it by default to all newly generated example integrations.